transport.go

```diff
@@ -346,7 +346,7 @@ func (p *connPool) roundTrip(ctx context.Context, req Request) (Response, error)
 		}
 	}()
 
-	var state = p.grabState()
+	state := p.grabState()
```

Comment:
Were these style changes done by a formatting tool?

Reply:
Yes, this is how https://github.com/mvdan/gofumpt formats.

The second commit fixes https://github.com/segmentio/kafka-go/issues/582 and slipped into this PR accidentally. But hopefully it is still fine...